test_sets/new_vehicle_data_hands_off_steering.txt

```diff
@@ -0,0 +1,33 @@
+./test_scripts/API/VehicleData/HandsOffSteering/003_GetVD_disallowed_after_PTU.lua
+./test_scripts/API/VehicleData/HandsOffSteering/004_GetVD_HMI_responds_with_invalid_data.lua
+./test_scripts/API/VehicleData/HandsOffSteering/005_GetVD_App_sends_invalid_request.lua
+./test_scripts/API/VehicleData/HandsOffSteering/006_SubscribeVD_Success.lua
+./test_scripts/API/VehicleData/HandsOffSteering/007_SubscribeVD_disallowed.lua
+./test_scripts/API/VehicleData/HandsOffSteering/008_SubscribeVD_disallowed_after_PTU.lua
+;./test_scripts/API/VehicleData/HandsOffSteering/009_SubscribeVD_not_success_codes_for_handsOffSteering_data.lua https://github.com/smartdevicelink/sdl_core/issues/3384
+./test_scripts/API/VehicleData/HandsOffSteering/010_SubscribeVD_HMI_responds_with_invalid_data.lua
+./test_scripts/API/VehicleData/HandsOffSteering/011_SubscribeVD_App_sends_invalid_request.lua
+./test_scripts/API/VehicleData/HandsOffSteering/012_SubscribeVD_for_two_Apps.lua
+./test_scripts/API/VehicleData/HandsOffSteering/013_UnsubscribeVD_Success.lua
+./test_scripts/API/VehicleData/HandsOffSteering/014_UnsubscribeVD_disallowed.lua
+./test_scripts/API/VehicleData/HandsOffSteering/015_UnsubscribeVD_disallowed_after_PTU.lua
+;./test_scripts/API/VehicleData/HandsOffSteering/016_UnsubscribeVD_not_success_codes_for_handsOffSteering_data.lua https://github.com/smartdevicelink/sdl_core/issues/3384
+./test_scripts/API/VehicleData/HandsOffSteering/017_UnsubscribeVD_HMI_responds_with_invalid_data.lua
+./test_scripts/API/VehicleData/HandsOffSteering/018_UnsubscribeVD_App_sends_invalid_request.lua
+./test_scripts/API/VehicleData/HandsOffSteering/019_UnsubscribeVD_for_two_Apps.lua
+./test_scripts/API/VehicleData/HandsOffSteering/020_OnVD_notification_Success.lua
+;./test_scripts/API/VehicleData/HandsOffSteering/021_OnVD_notification_disallowed.lua https://github.com/smartdevicelink/sdl_core/issues/3365
+;./test_scripts/API/VehicleData/HandsOffSteering/022_OnVD_disallowed_after_PTU.lua https://github.com/smartdevicelink/sdl_core/issues/3365
+./test_scripts/API/VehicleData/HandsOffSteering/023_OnVD_notification_with_invalid_data.lua
+./test_scripts/API/VehicleData/HandsOffSteering/024_OnVD_notification_App_is_not_subscribed.lua
+./test_scripts/API/VehicleData/HandsOffSteering/025_OnVD_notification_for_two_Apps.lua
+./test_scripts/API/VehicleData/HandsOffSteering/026_handsOffSteering_parameter_processing_in_case_app_version_is_greater_than_parameter_version.lua
+./test_scripts/API/VehicleData/HandsOffSteering/027_handsOffSteering_parameter_processing_in_case_app_version_is_less_than_parameter_version.lua
+./test_scripts/API/VehicleData/HandsOffSteering/028_SubscribeVD_during_resumption_unexpected_disconnect.lua
+./test_scripts/API/VehicleData/HandsOffSteering/029_SubscribeVD_during_resumption_IGN_OFF_ON.lua
+;./test_scripts/API/VehicleData/HandsOffSteering/030_SubscribeVD_during_resumption_unexpected_disconnect_for_two_Apps.lua https://github.com/smartdevicelink/sdl_core/issues/2283
```

Comment:
https://github.com/smartdevicelink/sdl_core/issues/2283 is fixed in develop, so this test can be uncommented

Reply:
@jacobkeeler Please find update in e500340